tests/unit/packages/utils/initialize.bats

```diff
@@ -48,20 +48,6 @@ function assert_debian_variables() {
   assert_equal "${INSTALLER_SUFFIX}" "${DEBIAN_INSTALLER_SUFFIX}"
 }
 
-@test "${TEST_SUITE_PREFIX}error function writes correct contents to stderr" {
-  exp="oh nose :("
-  run error "${exp}"
-  assert_equal "$status" 0
-  assert_output_contains "${output}" "${exp}"
-}
-
-@test "${TEST_SUITE_PREFIX}info function writes correct contents to stdout" {
-  exp="something or other"
-  run info "${exp}"
-  assert_equal "$status" 0
-  assert_output_contains "${output}" "${exp}"
-}
-
```

Comment:
These were just some leftover tests that I forgot to pull out into a separate file after I started refactoring this one a bit. I'm planning on using the same convention here as we do in our Python-based repositories:

where the test directory paths will line up with their source file under test (albeit with the `.bats` extension for test execution), up until the test files start to get too bloated in which case we can add a directory that aligns to the source file under test (`utils.sh` in this case) and within that directory we can have files for the individual functions being tested.

At the moment I've already got separate files for the bigger functions, but still have a `utils.bats` as a catch all for the other functions that only have one or two tests